onesignalclient/notification.py

```diff
@@ -50,26 +54,32 @@ def include_player_ids(self, value):
     def include_segments(self):
         return self._include_segments
 
-    # notification.include_segments = [Notification.SEGMENT_ALL]    
-    @include_segments.setter
-    def include_segments(self, value):
+    def _validate_include_segments(self, value):
         if self.mode != self.SEGMENTS_MODE:
             raise TypeError('Mode should be set to segment.')
 
         if not isinstance(value, list):
             raise TypeError('Value must be a list.')
         
-        if len(value) > 3:
-            raise TypeError('Invalid segment count! Segments available: SEGMENT_ALL, SEGMENT_ACTIVE_USERS, SEGMENT_INACTIVE_USERS')
+        if len(value) > len(self.SEGMENTS):
+            raise TypeError('Invalid segment count!')
         else:
             for item in value:
-                if item not isinstance(value, str):
+                if not isinstance(item, str):
                     raise TypeError('A segment should be str!')
                 if item not in self.SEGMENTS:
-                    # You should use at least one of the declared segments
-                    raise TypeError('Invalid segment added! Segments available: SEGMENT_ALL, SEGMENT_ACTIVE_USERS, SEGMENT_INACTIVE_USERS')
+                    raise TypeError('Invalid segment added! You should use at least one of the declared segments')
+        
```

Comment:
Can you remove these unwanted whitespaces?

Ref. 'W293:blank line contains whitespace'